charts/kube-prometheus-stack/values.yaml

```diff
@@ -2377,8 +2377,17 @@ prometheus-node-exporter:
     labels:
       jobLabel: node-exporter
 
+  ## Enable scraping via kubernetes-service-endpoints
```

Comment:
Please double check this.

The values are below `prometheus-node-exporter` and not related to kube-state-metrics